wurst/systems/spells/gatherer/Radars.wurst

```diff
@@ -305,26 +360,39 @@ function onCast(int itemId)
         range += MAP_MAGIC_RANGE_BONUS
 
     forItemsInMap() i ->
-        if i.getPos().distanceTo(casterPos) < range and i.getTypeId() == itemId and not areHostilesNearItem(owner, i)
-            //Vision
-            var vis = createVision(owner, i.getPos(), 128, true)
-            vis.start()
-            doAfter(REVEAL_DURATION) ->
-                vis.destr()
+        if i.getPos().distanceTo(casterPos) < range and i.getTypeId() == itemId and i.isPickupable()
+            if not itemToRadarDummy.has(i)
+                //Make a new dummy item structure to show through fog
+                let dummy = createUnit(players[PLAYER_NEUTRAL_PASSIVE], dummyDictionary.get(i.getTypeId()), i.getPos(), (270.0).asAngleDegrees())
+                dummy.setVertexColor(255, 255, 255, 0)
+                radarDummyDeath.registerDeathEvent(i)
+                itemToRadarDummy.put(i, dummy)
+                dummy.addAbility('Amrf')
```

Comment:
Why do you use Stormcrow Form to make it flying? It doesn't look like you set the height level anywhere.

Reply:
Thats was leftover code from the previous method we tried to make the dummies visible for localplayer. Updated